examples/sample-app/README.md

```diff
@@ -174,7 +175,7 @@ This section covers how to perform all the steps of building, deploying, and upd
 
 6. Login as `test-admin` using any password
 
-        $ oc login --certificate-authority=openshift.local.config/master/ca.crt
+        $ oc login --certificate-authority=openshift.local.config/master/ca.crt --config=openshift.local.config/master/admin.kubeconfig -u test-admin
```

Comment:
We don't want to encourage people to alter their cluster admin kubeconfig. I'd prefer this remain acting against a separate one